### PR TITLE
Choice modal

### PR DESCRIPTION
**Pull request by uma-a**
Modal appears every time game page is refreshed. Spanish flag has been set in this pull request as well.

### CODE REVIEW

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 1]
```diff
@@ -653,7 +672,23 @@ <h2>Current Score:
...
+var standard_button= document.getElementsByClassName("standard-button")[0];
+
+spanish_button.addEventListener('click', function(){
+   spanish_flag=1; <!--setting the Spanish flag to know which deck to render-->
+   choice_modal.classList.add("hide"); <!--once a deck has been selected we want to hide the modal-->
+   choice_modal_background.classList.add("hide");
+});
+
+standard_button.addEventListener('click', function(){
+   spanish_flag=0;
```
Comment (kierseym) [inline src/main/java/views/AcesUp/AcesUp.flt.html]: Does this spanish_flag get used somewhere else to actually choose the deck or have we not implemented that?

File: src/main/java/views/AcesUp/AcesUp.flt.html [hunk 2]
```diff
@@ -653,7 +672,23 @@ <h2>Current Score:
...
+   choice_modal.classList.add("hide");
+   choice_modal_background.classList.add("hide");
+});
```
Comment (kierseym) [inline src/main/java/views/AcesUp/AcesUp.flt.html]: Will we need to add to the help modal for the Spanish deck rules?

**Comment by kierseym** [comment]
This all looks really good, I like that when the choice is made the modal goes away, it also is a good idea so set a variable within the event listener for each button so we can use it later to tell the java which deck to use.

File: src/main/java/views/AcesUp/AcesUp.flt.html (hunk 2 above)
Comment (uma-a) [reply src/main/java/views/AcesUp/AcesUp.flt.html]: Yes, that is a good point. We will need to inform the user that the Spanish deck has fewer cards than the standard deck and has different game logic regarding the Jokers.

File: src/main/java/views/AcesUp/AcesUp.flt.html (hunk 1 above)
Comment (uma-a) [reply src/main/java/views/AcesUp/AcesUp.flt.html]: That part has not been implemented yet. Hopefully setting this flag will allow us to render the appropriate deck based on the user's selection.

**Pull request merged**